lib/travis/build/appliances/setup_filter.rb

```diff
@@ -63,7 +63,8 @@ def blocklist
         }
 
         def apply?
-          enabled? and secrets.any?
+          sh.echo 'Secrets are not currently filtered on Windows, please be careful', ansi: 'yellow' if windows?
```

Comment:
instead of 'please be careful' could we instead say 'for more information, please refer to our documentation <link>'

Reply:
just worked with @MariadeAnton to improve the message